### PR TITLE
Gives the Capture objective a better description.

### PR DESCRIPTION
**Pull request by Ikarrus**
Because what the fuck is a capture point?

### CODE REVIEW

**Comment by Incoming5643** [comment]
Last I heard there were some real problems with the capture objective. The only way to initiate it is from a right click verb (on the target, not yourself), people can just walk out of the net, and if they later kill themselves/die in containment you don't get the point(s) for the capture.

Also it incentivizes throwing players into a small room with no escape with nothing to do.

**Comment by freerealestate** [comment]
What the fuck is a rare specimen?

**Comment by Anonus** [comment]
Aliens. Also, the net actually properly got changed to trap the people you use it on but I don't think anyone noticed.

File: code/datums/mind.dm [hunk 1]
```diff
@@ -710,7 +710,7 @@
 						new_objective.explanation_text = "Download [target_number] research levels."
 					if("capture")
 						new_objective = new /datum/objective/capture
-						new_objective.explanation_text = "Accumulate [target_number] capture points."
+						new_objective.explanation_text = "Capture [target_number] lifeforms with an energy net. Live, rare specimen are worth more."
```
Comment (RemieRichards) [inline code/datums/mind.dm]: the plural of specimen is specimens.

File: code/game/gamemodes/objective.dm [hunk 2]
```diff
@@ -499,7 +499,7 @@ var/global/list/possible_items_special = list()
 
 /datum/objective/capture/proc/gen_amount_goal()
 		target_amount = rand(5,10)
-		explanation_text = "Accumulate [target_amount] capture point\s. It is better if they remain relatively unharmed."
+		explanation_text = "Capture [target_amount] lifeform\s with an energy net. Live, rare specimen are worth more."
```
Comment (RemieRichards) [inline code/game/gamemodes/objective.dm]: see above.

**Pull request merged**